lib/dcv/solr/document_adapter/active_fedora.rb

```diff
@@ -0,0 +1,213 @@
+# -*- encoding : utf-8 -*-
+module Dcv::Solr::DocumentAdapter
+  class << self
+    # Create a new adapter object
+    # @param obj [ActiveFedora::Base]
+    # @return [Dcv::Solr::DocumentAdapter::ActiveFedora]
+    def ActiveFedora obj
```

Comment:
Should this method name be snake_case?

Reply:
No - this is akin to `Nokogiri::XML(...)` or `Array(...)`

Reply:
Ah okay!

Reply:
This hides the registry of adapter types from the client code.